backoffice/src/Helpers/MetricsHelper.cs

```diff
@@ -33,6 +33,24 @@ public static class MetricsHelper
 
         return groupedDroits;
     }
+  
+    public static IEnumerable<object>? GetClosedDroitsMetrics(List<Droit> droits)
+    {
+        var allStatuses = Enum.GetValues(typeof(DroitStatus)).Cast<DroitStatus>().ToList();
+
+        // Find the earliest and latest years
+        var minYear = droits.Min(d => d.ClosedDate?.Year) ?? 1990;
```

Comment:
Probably not safe having this as 1990, Maybe try DateTime.Min

Reply:
This is done now.